lib/csp.js

```diff
@@ -27,6 +29,22 @@ module.exports = function (options) {
     }
 
     return function csp(req, res, next) {
+        if (styleNonce) {
```

Comment:
meh, it hurts the readability imo.  also forces the res object to be passed around more than is necessary